docs/api/inputs/chat.md

```diff
@@ -258,3 +258,83 @@ configuration with a dictionary conforming to the config.
 ```{eval-rst}
 .. autoclass:: marimo.ai.ChatAttachment
 ```
+
+## Supported Model Providers
+
+You can refer to the [LiteLLM page](https://litellm.vercel.app/docs/providers) for a list of OpenAI API supported providers. If you want any specific provider added explicitly (ones that don't abide by the standard OpenAI API format), you can file a [feature request](https://github.com/marimo-team/marimo/issues/new?template=feature_request.yaml).
+
+Normally, overriding the `base_url` parameter should work. Here are some examples:
+
+::::{tab-set}
+:::{tab-item} Cerebras
+
+```python
+chatbot = mo.ui.chat(
+   mo.ai.llm.openai(
+       model="llama3.1-8b",
+       api_key="csk-...", # insert your key here,
+       base_url="https://api.cerebras.ai/v1/",
+   ),
+)
+chatbot
+```
+
+:::
+:::{tab-item} Groq
+
+```python
+chatbot = mo.ui.chat(
+   mo.ai.llm.openai(
+       model="llama-3.1-70b-versatile",
+       api_key="gsk_...", # insert your key here,
+       base_url="https://api.groq.com/openai/v1/",
+   ),
+)
+chatbot
+```
+
+:::
+::::
+
+```{note}
+We have added examples for GROQ and Cerebras. These providers offer free API keys and are great for trying out Llama models (from Meta). You can sign up on their platforms and integrate with various AI integrations in marimo easily. For more information, refer to the [AI completion in marimo](/guides/editor_features/ai_completion#ai-completion).
+```
+
+### Finding the Right Endpoint URLs
+
+While the [LiteLLM page](https://litellm.vercel.app/docs/providers) provides a list of OpenAI API supported providers, the endpoints mentioned there are for chat completion and are not suitable for the `base_url` parameter. You need to do your research on the documentation websites of the model providers you select and find the right endpoint URL for normal inferencing (can usually be found in the curl commands).
+
+For example, here is how you can override the `base_url` and specify the endpoint for Cerebras:
+
+```python
+chatbot = mo.ui.chat(
+   mo.ai.llm.openai(
+       model="llama3.1-8b",
+       api_key=key,
+       base_url="https://api.cerebras.ai/v1/",
+   ),
+)
+chatbot
+```
+
+If you want more examples for model providers and their endpoints listed, please submit a [feature request](https://github.com/marimo-team/marimo/issues/new?template=documentation.yaml).
+
+### Example Usage
```

Comment:
i think we can delete these, we already have the same example above

Reply:
Ah alright; my reasoning was, the first link above (is for a feature request) to include model providers which can't be run through overriding the `base_url` -> not openAI compatible; the hyperlink below is for a doc change (.doc.yaml) if people want more model providers to be listed in this chat.md doc webpage (for reference).